bundles/ranvier-commands/commands/get.js

```diff
@@ -63,6 +63,7 @@ module.exports = (srcPath) => {
       player.addItem(item);
 
       Broadcast.sayAt(player, `<green>You receive loot: </green>${item.display}<green>.</green>`);
+      Broadcast.sayAt(room, `<green>${player.name} receives loot: </green>${item.display}<green>.</green>`)
```

Comment:
Removed anything that has potential for abuse, and added broadcasting for certain skills.